pom.xml

```diff
@@ -114,6 +114,7 @@
         <grpc.version>1.26.0</grpc.version>
         <protobuf.version>3.11.3</protobuf.version>
         <picocli.version>3.9.0</picocli.version>
+        <classgraph.version>4.8.62</classgraph.version>
```

Comment:
We should also use this version in `hazelcast-jet-core/pom.xml` for `classgraph` instead of hard-coded version as it is now.

Reply:
yes forgot to change it, will update.